tarantool/mesh_connection.py

```diff
@@ -25,21 +35,111 @@ def getnext(self):
         return self.addrs[tmp]
 
 
+def parse_uri(uri_str):
```

Comment:
Just observed that we have the function `warn` from `tarantool.error` and use it to emit warnins. So it worth to give a user information about validation errors we ignore during cluster discovery. Sorry, I don't know about that before. It expands the scope of this issue, but I guess not really much.

(Provide an ability to pass custom logger for Connection to, say, filter or silent warnigns is the separate feature and I'll file an issue about that.)

Reply:
Done